Library/Formula/cv.rb

```diff
@@ -0,0 +1,15 @@
+class Cv < Formula
+  desc "Coreutils Viewer: Show progress for coreutils"
+  homepage "https://github.com/BestPig/cv"
```

Comment:
OK I have requested the author to create a new release supporting OSX.
see Xfennec/cv#45